nautobot/docs/installation/nautobot.md

```diff
@@ -172,6 +172,19 @@ To use remote file storage, add `django-storages` to your `local_requirements.tx
 (nautobot) $ echo django-storages >> /opt/nautobot/local_requirements.txt
 ```
 
+## Check your Configuration
```

Comment:
This seems a little early to me - should it maybe instead be moved down to near the "Test the Application" heading?

Reply:
That's fair.